example/app/css/app.css

```diff
@@ -4,6 +4,138 @@ html, body, #canvas {
   padding: 0;
   margin: 0;
 }
+button {
+  border: none;
+  background: none;
+}
+
+/* screen interaction */
+@font-face {font-family: "Font Awesome 5 Free Solid";
+  src: url("../assets/font-awesome-5/29f589f173dcc69ef6c805b711894998.eot"); /* IE9*/
+  src: url("../assets/font-awesome-5/29f589f173dcc69ef6c805b711894998.eot?#iefix") format("embedded-opentype"), /* IE6-IE8 */
+  url("../assets/font-awesome-5/29f589f173dcc69ef6c805b711894998.woff2") format("woff2"), /* chrome、firefox */
+  url("../assets/font-awesome-5/29f589f173dcc69ef6c805b711894998.woff") format("woff"), /* chrome、firefox */
+  url("../assets/font-awesome-5/29f589f173dcc69ef6c805b711894998.ttf") format("truetype"), /* chrome、firefox、opera、Safari, Android, iOS 4.2+*/
+  url("../assets/font-awesome-5/29f589f173dcc69ef6c805b711894998.svg#Font Awesome 5 Free Solid") format("svg"); /* iOS 4.1- */
+}
+
+.io-editing-tools {
```

Comment:
I think we should align the button styles with the existing ones:

![image](https://user-images.githubusercontent.com/9433996/83745712-41d77f80-a65e-11ea-9d96-cf3f4b3e7264.png)

cf. https://github.com/pinussilvestrus/postit-js/blob/master/example/app/css/app.css#L26

Reply:
Great idea! By align, do you mean style (shadow, height?) or pull the new buttons down?

Reply:
yep, exactly